torchao/experimental/ops/mps/test/test_quantizer.py

```diff
@@ -0,0 +1,104 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the license found in the
+# LICENSE file in the root directory of this source tree.
+
+from typing import Optional
+import copy
+import os
+import sys
+
+import torch
+import torchao_mps_ops
+import unittest
+
+torchao_root: Optional[str] = os.getenv("TORCHAO_ROOT")
+assert torchao_root is not None, "TORCHAO_ROOT is not set"
+
+sys.path.insert(0, torchao_root)
```

Comment:
This look like a debug code and should be removed...

Reply:
done!